pages/api/pairs/index.tsx

```diff
@@ -1,6 +1,6 @@
 import { Network } from "types/network";
 import { NextApiRequest, NextApiResponse } from "next";
-import { buildPoolsInfo } from "utils/info/pools";
+import { buildPoolsInfo, buildPoolsInfoV2 } from "utils/info/pools";
```

Comment:
what happens with buildPoolsInfo?

Reply:
nothing, i added a new buildPoolsInfo with my code to have no impact or reverse easily